### PR TITLE
DEX-1206 Billing Address Capture onto SDK v2

### PR DESCRIPTION
**Pull request by d4r1091**
# What this PR does

### Based on "flow removal (`feature/DEX-1968-Remove-flow`)" branch

- Ported billing address capture work onto PrimerSDKv2. From -> https://github.com/primer-io/primer-sdk-ios/pull/385
- Bringing in also a major Strings update, tackled here -> https://github.com/primer-io/primer-sdk-ios/pull/384
- Removed reference to `ZipCode` as per a cross-platform alignment with Web and Android
- Updated `en` Localizable.strings
- Minor codebase improvements

### CODE REVIEW

File: Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultComponentView.swift [hunk 1]
```diff
@@ -27,36 +27,35 @@ class PrimerResultComponentView: PrimerView {
...
         
         if let imageView = imageView {
-            stackView.addArrangedSubview(imageView)
+            verticalStackView.addArrangedSubview(imageView)
         }
         
         if let message = message {
             let messageLabel = UILabel()
             messageLabel.text = message
-            messageLabel.accessibilityIdentifier = "result_component_view_message_label"
```
Comment (EvansPie) [inline Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultComponentView.swift:50]: Don't we need this for UI tests?

File: Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/Fields/AddressField.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        let addressLine1FieldView = PrimerAddressLine1FieldView()
+        addressLine1FieldView.placeholder = Strings.CardFormView.AddressLine1.placeholder
+        addressLine1FieldView.heightAnchor.constraint(equalToConstant: 36).isActive = true
+        addressLine1FieldView.textColor = theme.input.text.color
+        addressLine1FieldView.delegate = delegate
+        return addressLine1FieldView
+    }
+}
+
+class PrimerAddressLine2Field: PrimerAddressField {
```
Comment (EvansPie) [inline Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/Fields/AddressField.swift:38]: Is there any difference between `PrimerAddressLine1Field` and `PrimerAddressLine2Field`? Can't the `PrimerAddressLine1Field` be renamed to `PrimerAddressLineField` and be reused?

**Review by EvansPie — APPROVED** [review]

File: Sources/PrimerSDK/Classes/Data Models/CountryCode.swift [hunk 3]
```diff
@@ -525,24 +275,49 @@ internal extension CountryCode {
...
-            return "fr-BE"
-        case .es:
-            return "es-ES"
-        case .it:
-            return "it-IT"
-        default:
-            return "n/a"
+    
+    private static var languageCode: String {
+        Locale.current.languageCode ?? "en"
```
[the remark this answers is not in the document]
Comment (d4r1091) [reply Sources/PrimerSDK/Classes/Data Models/CountryCode.swift:280]: @EvansPie will post the discussion here:

----------------------------------------------------------------
That `var` is used in `CountryCode.swift:312` to retrieve the local JSON config matching the system language set on the device at the moment.
Is there any other way the merchant can specify a language that differs from the device language and notify our SDK?
```
        guard let localizedCountriesData = jsonParser.loadJsonData(fileName: CountryCode.languageCode) else {
            return nil
        }
```
----------------------------------------------------------------

Your reply:

Yeah there is, via the PrimerSettings. But it doesn't apply on this case.

File: Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultComponentView.swift (hunk 1 above)
Comment (d4r1091) [reply Sources/PrimerSDK/Classes/User Interface/Components/PrimerResultComponentView.swift:50]: We rely on the image now but I'll add it back - thanks for catching it!

File: Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/Fields/AddressField.swift (hunk 2 above)
Comment (d4r1091) [reply Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/FormsTokenizationViewModel/Fields/AddressField.swift:38]: `PrimerAddressLine2Field` contains a different validation, and other analytics fields etc...other than that nothing much.
Though I wanted to make it two different fields as we make a "class compare" in `CardFormPaymentMethodTokenizationViewModel.swift:869` so I didn't want to disrupt the logic

**Pull request merged**